src/Tools/Md2Html/Md2Html.csproj

```diff
@@ -1,61 +1,72 @@
-    </PackageReference>
-    <PackageReference Include="ILMerge">
-      <Version>3.0.41</Version>
-    </PackageReference>
-    <PackageReference Include="Markdig">
-      <Version>0.22.0</Version>
-    </PackageReference>
-    <PackageReference Include="NDesk.Options">
-      <Version>0.2.1</Version>
-    </PackageReference>
-  </ItemGroup>
-  <Target Name="AfterBuildOps" AfterTargets="Build">
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="if not exist $(SharedOutputPath) mkdir $(SharedOutputPath)"/>
-    <!-- the ILMergePath property points to the location of ILMerge.exe console application -->
-    <Exec Condition=" '$([MSBuild]::IsOSPlatform(`Windows`))' == 'True'"
-		Command="$(ILMergeConsolePath) $(OutputPath)Md2Html.exe ^
+<Project Sdk="Microsoft.NET.Sdk">
+    <ImportGroup Label="PropertySheets">
+        <Import Project="$(SolutionDir)Config\CS_SDK.props" />
+    </ImportGroup>
+    <PropertyGroup>
+        <ProjectGuid>{0893F745-CB1A-427A-8E87-CA927273039A}</ProjectGuid>
+        <OutputType>Exe</OutputType>
+        <RootNamespace>Md2Html</RootNamespace>
+        <AssemblyName>Md2Html</AssemblyName>
+        <SharedOutputPath>$(OutputPath)\Md2Html\</SharedOutputPath>
+        <OutputPath>bin\$(Platform)\$(Configuration)\</OutputPath>
+    </PropertyGroup>
+    <PropertyGroup Condition=" '$(TargetFramework)' == 'net6.0' ">
+        <PublishSingleFile>true</PublishSingleFile>
+        <SelfContained>false</SelfContained>
+    </PropertyGroup>
+    <ItemGroup Condition=" '$(TargetFramework)' == 'net48' ">
+        <Reference Include="System" />
+        <Reference Include="System.Core" />
+        <Reference Include="System.Xml.Linq" />
+        <Reference Include="System.Data.DataSetExtensions" />
+        <Reference Include="Microsoft.CSharp" />
+        <Reference Include="System.Data" />
+        <Reference Include="System.Net.Http" />
+        <Reference Include="System.Xml" />
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="AngleSharp" Version="0.14.0" />
+        <PackageReference Include="AngleSharp.Css" Version="0.14.2" />
+        <PackageReference Include="CommandLineParser">
+            <Version>2.8.0</Version>
+        </PackageReference> 
+        <PackageReference Include="HtmlSanitizer">
+            <Version>5.0.372</Version>
+        </PackageReference>
+        <PackageReference Include="ILMerge" Condition=" '$(TargetFramework)' == 'net48'">
+            <Version>3.0.41</Version>
+        </PackageReference>
+        <PackageReference Include="Markdig">
+            <Version>0.22.0</Version>
+        </PackageReference>
+        <PackageReference Include="System.Buffers" Version="4.5.1" />
+        <PackageReference Include="System.Memory" Version="4.5.5" />
+        <PackageReference Include="System.Numerics.Vectors" Version="4.5.0" />
+        <PackageReference Include="System.Runtime.CompilerServices.Unsafe" Version="6.0.0" />
```

Comment:
why do we need direct references to these?

Reply:
all these references need to be bundled in a single executable. From what I can tell, these dependencies were needed for net48 (when using ilmerge). I assume they are still needed on net6 (I did not check though)

Reply:
Oh, I see. These are actually transitive dependencies from one of the other packageRefs.
I can remove these